namer/namer.cc

```diff
@@ -1432,6 +1483,14 @@ class SymbolDefiner {
         // all the `nonDeletableDefinitions` from all files get defined, then delete all the old
         // methods, then define all the methods.
         if (oldFoundHashes.has_value()) {
+            for (const auto &oldFieldHash : oldFoundHashes.value().fieldHashes) {
+                if (oldFieldHash.owner.isInstanceVariable) {
+                    deleteViaFullNameHash(ctx, oldFieldHash);
+                }
+            }
```

Comment:
another thought: should we attempt to put `fieldHashes` and `methodHashes` into the same vector on oldFoundHashes, so that we can just have one loop here, and then make `deleteViaFullNameHash` know how to operate on two different kinds of symbols? not sure what feels best, especially considering that we will add type members and static fields ~soon

Reply:
Same as other places in this PR: no super-strong opinion feeling, slight preference for keeping everything separate, but annoyance at how doing so requires updating a zillion places.